src/contexts/ConfigurationContext.tsx

```diff
@@ -476,6 +475,23 @@ export const ConfigContextProvider = ({ children }: Props) => {
       );
     }
 
+    const isRankerEnabled = (rerankerName: string | undefined) => {
+      return rerankerName === "default" || rerankerName === "slingshot"
+        || rerankerName === "mmr" || false
+    }
+    const getRerankerId = (rerankerName: string | undefined) => {
+      if (rerankerName === "mmr")  return mmr_reranker_id
+      else if (rerankerName === "slingshot") return slingshot_reranker_id
+      else return normal_reranker_id
```

Comment:
I think the name "default" does not make sense to me in the reranker name. We should figure out it's name.
Default seems to indicate it's used by default, and that is not true in this case. If unspecified then it's going to use what is called here "normal" reranker.

Reply:
Yeah that makes sense.